QMLFileType.js

```diff
@@ -92,9 +93,11 @@ QMLFileType.prototype.write = function(translations, locales) {
         // for each extracted string, write out the translations of it
         translationLocales.forEach(function(locale) {
             logger.trace("Localizing QML strings to " + locale);
+            var manipulateKey = res.cleanHashKeyForTranslation(locale).replace(res.getContext(),"");
```

Comment:
Why did you delete the context from the hash for the context string? I think that is the whole point for using a context string instead of just a regular string...

Reply:
When the loctool parse the extracted string, the hashkey is something like below:
 ` crs_app1_Volume_ko-KR_hello_x-qml_, `
(Volume is context value, hello is source string)

But, When the parsing xliff files, It stores a value with key, but at this time xliff file doesn't have context value.

So matched target string's key is
'crs_app1__ko-KR_hello'

So In order to find proper translated value from xliff, the key shouldn't contain context value. If not, I can't get the correct translated string.

Reply:
Do you mean that the current LG xliffs don't have this context? If not, maybe you make the code check for the string with the context, and if it is not there, check for it without the context.

Reply:
Yes, LG xliffs don't have context name. (Ih has only source, target, and comment). I updated code to check two times to get a translation. One with a full key, another is key without context.